test/cli/pubsub.js

```diff
@@ -14,7 +14,7 @@ const createTempNode = ''
 const repoPath = require('./index').repoPath
```

Comment:
@richardschneider are these tests supposed to be enabled?

Reply:
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/1211152/33123027-d6ac704c-cf70-11e7-8e2b-40bcedef150f.png">

Reply:
I did comment somewhere, that the test is very old and doesn't really work,  By doing `require('./index').repoPath`, it brings in all the other tests.